pimd/pim_nht.c

```diff
@@ -975,6 +1025,19 @@ int pim_ecmp_nexthop_lookup(struct pim_instance *pim,
 			i++;
 			continue;
 		}
+
+		if (!pim_ifp->pim_enable) {
```

Comment:
Please add this as an OR condition in the previous check. It's doing the same thing.